source/clear-linux/tutorials/tutorials.rst

```diff
@@ -3,12 +3,14 @@
 Tutorials
 #########
 
+This section provides instruction to walk you through completing |CL| use
```

Comment:
Consider changing to: This section provides detailed instructions to guide you through completing specific |CLOSIA| use cases.